src/main/java/com/commercetools/sync/utils/SphereClientUtils.java

```diff
@@ -0,0 +1,55 @@
+package com.commercetools.sync.utils;
+
+import static java.lang.String.format;
+
+import com.commercetools.sync.commons.utils.ClientConfigurationUtils;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.client.SphereClientConfig;
+import java.io.InputStream;
+import java.util.Properties;
+import javax.annotation.Nonnull;
+
+public final class SphereClientUtils {
```

Comment:
It looks, we have same logic in sync project, can we change this in future to prevent duplication. wdyt ?

Reply:
yes, the logic in the sync project is used for the integration tests there only. It's not yet meant to be exposed. But it will be exposed later for usage by the library: https://github.com/commercetools/commercetools-sync-java/issues/144

Reply:
Good, thanks for the explanation!